### ISSUE TITLE
[Request] TODO Logo Usage for OSPO On Ramp

### ISSUE THREAD

**Issue by anajsana** [todogroup/governance#147]
## Request

- [ ] Use TODO Logo in OSPO On Ramp landing page
- [ ] Promote OSPO On Ramp project in main TODO SM channels

## Description

OSPO On-Ramp project has been moving forward since September, initiated by some of the TODO Participants from the EU Chapter + OSPO Zone Participants. Last Friday, OSPO On-Ramp landing page was published: https://ospo.zone/onramp/

The request from TODO EU Chapter participants was to have a common place to bring open source/OSPO 101 discussions, with a neutral mailing list and landing page to share common resources (including TODO resources). 

Please note that:

* Moderators from this meeting are organized by OSPO Zone's MP and TODO Group's PM.
* Speakers include TODO Members willing to share their knowledge there.
* It is an EMEA-friendly timezone, monthly meeting, and with very basic topics on how to start an OSPO or start in the open source journey.

## Closing Conditions

- [ ] SC Approves/Declines this request

**Comment by geekygirldawn** [comment]
I think there is a bigger question that needs to be resolved before we approve a logo for use with this meeting. 

OSPO Zone is an [OSPO Alliance effort](https://ospo.zone/about/), so it seems like we should clarify what kind of relationship the TODO Group has with the OSPO Alliance first?

**Comment by anajsana** [comment]
> I think there is a bigger question that needs to be resolved before we approve a logo for use with this meeting.
> 
> OSPO Zone is an [OSPO Alliance effort](https://ospo.zone/about/), so it seems like we should clarify what kind of relationship the TODO Group has with the OSPO Alliance first?

You are right @geekygirldawn. However, I would like to clarify that OSPO On Ramp is not intended to be a OSPO Zone project, but a cross-community project where both TODO and OSPO Zone collaborate (or at least, that was the initial idea). That's one of the reasons why the mailing list used is a brand new mailing list (not the OSPO Zone ML) and everyone can post content on it).

So, the way I see this:

1) I agree that we should clarify the relationship with OSPO Alliance by opening in a new issue or SC Meeting
2) OSPO On-Ramp should be treated as an independent project, and has nothing to do with OSPO Alliance's relationship

**Comment by caniszczyk** [comment]
I share the same sentiments with @geekygirldawn, it's still an early effort and we should wait and just go through the process for branding usage (which may take some time). We can attend meetings to check it out though!

There is some overlap with https://github.com/todogroup/ospology here which is meant for OSPOs to gather / share stories in a public fashion.

**Comment by anajsana** [comment]
> I share the same sentiments with @geekygirldawn, it's still an early effort and we should wait and just go through the process for branding usage (which may take some time). We can attend meetings to check it out though!
> 

Ok, let's do that if people see it this way.

> There is some overlap with https://github.com/todogroup/ospology here which is meant for OSPOs to gather / share stories in a public fashion.

IMHO: OSPO On Ramp meetings are more EMEA Friendly than the OSPOlogy sessions. Some people based in Europe has just ended their working hours at 5:00pm. OSPO On Ramp (which starts at 10:30am CET) tries to fill that gap that the TODO Group (at least for now) is not covering.

Despite the fact that we have the TODO EU Chapter meetings at 3:00pm CET,  these are (at least until now) different to what OSPOlogy is. These meetings are more focused on reviewing things that happened in the OSPO ecosystem, TODO News and some broader discussions.

**Comment by vmbrasseur** [comment]
Hey there, team. Are we still blocked on this? If so, what's the blocker? If it's merely "wait and see" then I'd suggest it's "on hold" rather than "blocked."

Thoughts?

**Comment by anajsana** [comment]
This is an outdated task and don't think is going to be unblocked soon + a lot of new things have happened in both communities.

Closing this. if someone would like to re-take this topic, please create a new issue with reasoning 🙂
